assets/queries/terraform/aws/ebs_default_encryption_disabled/metadata.json

```
@@ -1,7 +1,7 @@
 {
   "id": "3d3f6270-546b-443c-adb4-bb6fb2187ca6",
   "queryName": "EBS Default Encryption Disabled",
-  "severity": "HIGH",
+  "severity": "MEDIUM",
```

```suggestion
  "severity": "HIGH",
```
nevermind about my previous suggestion let's keep this at HIGH